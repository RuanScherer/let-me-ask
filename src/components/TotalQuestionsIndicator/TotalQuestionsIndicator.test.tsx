import { render, screen } from '@testing-library/react';
import { TotalQuestionsIndicator } from './';

describe('Total Questions Indicator Component', () => {
	test('Show questions count', () => {
		render(<TotalQuestionsIndicator questionsCount={3} />);

		expect(screen.getByText(/3/)).toBeInTheDocument();
	});

	test('Snapshot must be always the same', () => {
		const { container } = render(<TotalQuestionsIndicator questionsCount={5} />);

		expect(container.firstChild).toMatchSnapshot();
	});
});

export { };
