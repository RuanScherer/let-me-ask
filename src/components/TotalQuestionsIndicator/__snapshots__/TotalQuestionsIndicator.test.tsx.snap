// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Total Questions Indicator Component Snapshot must be always the same 1`] = `
<span
  class="sc-bdnxRM fVHjYQ"
>
  5
   pergunta(s)
</span>
`;
